internal/drand-cli/dkg_cli.go

```diff
@@ -559,8 +559,18 @@ func convert(entry *drand.DKGEntry) printModel {
 
 		b := strings.Builder{}
 		b.WriteString("[")
+
+		checkmark := ""
 		for _, a := range arr {
-			b.WriteString(fmt.Sprintf("\n\t%s,", a.Address))
+			if checks && util.Contains(entry.Remaining, a) {
+				checkmark = " ☐ "
+				if util.Contains(entry.Acceptors, a) {
+					checkmark = " ☑ "
+				} else if util.Contains(entry.Rejectors, a) {
+					checkmark = " ☒ "
+				}
+			}
+			b.WriteString(fmt.Sprintf("\n\t%s%s,", a.Address, checkmark))
```

Comment:
might be best to put the checkmark first or inconsistent address length may make it hard to read